### PR TITLE
Chapter 2

### PR DESCRIPTION
**Pull request by ShariaHabib**
Changed as per instruction

### CODE REVIEW

File: chapter-2/chapter2.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Overview of Java
+### *Priciples of OOP*
+
+1. Encapsulation: Mechanism that binds code and data together to keep safe from outside interference an misuse.
```
Comment (towfiq-ul) [inline chapter-2/chapter2.md]: rename the file name from **chapter2.md** to **README.md**

**Review by towfiq-ul — CHANGES_REQUESTED** [review]
Change the **MD** file name

File: chapter-2/chapter2.md (hunk 1 above)
Comment (ShariaHabib) [reply chapter-2/chapter2.md]: Done vaiya.

**Review by towfiq-ul — APPROVED** [review]
LGTM

**Pull request merged**